CMakeLists.txt

```
@@ -1978,7 +1978,43 @@ find_package(Vorbis REQUIRED)
 target_link_libraries(mixxx-lib PRIVATE Vorbis::vorbis Vorbis::vorbisenc Vorbis::vorbisfile)
 
 # PortAudio
-find_package(PortAudio REQUIRED)
+if(NOT APPLE)
+  find_package(PortAudio REQUIRED)
+else()
+  # The macOS build environment contains a portaudio 19.6 without macOS Ventura 13.0 Support
+  set(PortAudio_VERSION 19.7.0)
+  set(PortAudio_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib/portaudio-install")
+  set(PortAudio_LIBRARIES "${PortAudio_INSTALL_DIR}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}portaudio${CMAKE_STATIC_LIBRARY_SUFFIX}")
+  set(PortAudio_INCLUDE_DIRS "${PortAudio_INSTALL_DIR}/include")
+
+  ExternalProject_Add(portaudio
+    URL "https://github.com/PortAudio/portaudio/archive/refs/tags/v${PortAudio_VERSION}.zip"
+    URL_HASH SHA256=ce1e7b27ad362cb9745de7da3d266a996b085ca75a12ac62c881f88ab6894acf
```

why not use ExternalProject's Git features? 
```suggestion
    GIT_REPOSITORY "https://github.com/PortAudio/portaudio/"
    GIT_TAG "v${PortAudio_VERSION}"
    GIT_SHALLOW true
```

---

Because it works without git installed, from a tarball for instance. 
It is also in a good companion of libkeyfinder where we choose the same method. 
And not at least it already works.

---

Mhmm, well its less brittle than relying on the download url I'd argue, but its fine either way.